src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -616,6 +564,36 @@ function getColumnOffsetForSelection(type: MarkdownButtonType, nothingSelected:
 	}
 }
 
+/**
+ * When markdown is already formatted correctly and doesn't need transformed, insert markdown based on current editor selection
+ * @param markdownToInsert formatted markdown
+ * @param editorControl editor control for cell
+ */
+export async function insertFormattedMarkdown(markdownToInsert: string, editorControl: IEditor): Promise<void> {
```

Comment:
Should be `editorControl` since you handle empty editor controls (is that actually a valid scenario though?)

Reply:
I don't understand this comment. Are you missing a `?`? 

And no, that's not a valid scenario; to insert markdown, we need a valid editor control.

Reply:
Oops, yeah that's supposed to be `editorControl?`

Seems like you could get rid of the if check then? (oh how I wish we had strict typing in all the core...)

Reply:
Ok actually looked at the code - it seems like you do actually expect to occasionally get an undefined editor. So in that case you should make the parameter `editorControl?`